src/main/java/com/jfrog/ide/idea/scan/ScanBinaryExecutor.java

```diff
@@ -32,16 +36,37 @@ public abstract class ScanBinaryExecutor {
     protected List<String> supportedLanguages;
     private final boolean shouldExecute;
 
+    private static final String ENV_PLATFORM = "JF_PLATFORM_URL";
+    private static final String ENV_USER = "JF_USER";
+    private static final String ENV_PASSWORD = "JF_PASS";
+    private static final String ENV_ACCESS_TOKEN = "JF_TOKEN";
+
+
     ScanBinaryExecutor(String scanType, String binaryName) {
         this.scanType = scanType;
         if (SystemUtils.IS_OS_WINDOWS) {
             binaryName += ".exe";
         }
         Path binaryPath = BINARIES_DIR.resolve(binaryName);
-        commandExecutor = new CommandExecutor(binaryPath.toString(), Maps.newHashMap());
+        commandExecutor = new CommandExecutor(binaryPath.toString(), creatEnvWithCredentials());
         shouldExecute = Files.exists(binaryPath);
     }
 
+    private Map<String, String> creatEnvWithCredentials() {
+        Map<String, String> env = new HashMap<>();
+        ServerConfigImpl serverConfig = GlobalSettings.getInstance().getServerConfig();
+        if (serverConfig.isXrayConfigured()) {
+            env.put(ENV_PLATFORM, serverConfig.getUrl());
+            if (StringUtils.isNotEmpty(serverConfig.getAccessToken())) {
+                env.put(ENV_ACCESS_TOKEN, serverConfig.getAccessToken());
+            } else {
+                env.put(ENV_USER, serverConfig.getUsername());
+                env.put(ENV_PASSWORD, serverConfig.getPassword());
+            }
```

Comment:
We may also consider configuring the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables. The information can be extracted from the user's configured proxy:
```java
ProxyConfiguration proxyConfiguration = serverConfig.getProxyConfForTargetUrl(serverConfig.getUrl());
```